prac_05/kivy_layout.kv

```diff
@@ -0,0 +1,20 @@
+# This is a Kv file.
+# It contains the declarations of the widgets that will be displayed.
+BoxLayout:
+    orientation: 'vertical'
+    Label:
+        text: app.status_text
+        font_size: 60
+        size_hint_y: 0.2
```

Comment:
One of the instructions are to change something from 20% to 10%. This is yet to be done.